rest-service/manager_rest/resource_manager.py

```diff
@@ -1081,17 +1081,24 @@ def check_for_executions(self, execution, force, queue):
             return system_exec_running or execution_running
 
     def _check_for_active_executions(self, execution, queue):
+        if execution.created_at is not None:
+            status_filter = sql_or(
+                modelsExecution.status.in_(ExecutionState.ACTIVE_STATES),
```

Comment:
modelsExecution - should this be models.Execution?

Reply:
yeah some later commit fixes this, sorry. I thought i caught all of these in the rebase, but apparently not.